packages/moonstone/VideoPlayer/MediaSliderDecorator.js

```diff
@@ -0,0 +1,174 @@
+import {adaptEvent, handle, forKey, forward, oneOf, preventDefault} from '@enact/core/handle';
+import hoc from '@enact/core/hoc';
+import {calcPercent} from '@enact/ui/Slider/utils';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import MediaKnob from './MediaKnob';
+
+// decrements the MediaKnob position if we're tracking
+const decrement = (state) => {
+	if (state.tracking && state.x > 0) {
+		const x = Math.max(0, state.x - 0.05);
```

Comment:
I think it's irrelevant in this case but also not feasible to do given the current encapsulation. MediaSliderDecorator wraps around MediaSlider but MediaSlider is the component setting `knobStep`. In practice, that setting isn't used because `knobStep` is only used when activating the knob and then pressing 5-way controls which isn't supported by MediaSlider.

Going to remove the prop from MediaSlider and keep this hard coded value for now.